feature-utils/poly-look/src/react-components/buttons/scrollButton.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
```

Comment:
If possible, also some tests here could help. I can help you with that, if you want.

Reply:
I was already planning to do it 😄